.github/workflows/release.yml

```diff
@@ -20,20 +25,21 @@ jobs:
       github.repository == 'remix-run/remix' &&
       !contains(github.ref, 'nightly')
     runs-on: ubuntu-latest
+    timeout-minutes: 2
```

Comment:
For this job, I don't think 2 minutes is enough, but core maintainers can confirm.
Usually releases are carefully monitored, so I don't think there is a real use case to this limit.

Reply:
I agree - a quick peek at past runs shows we've had a few jobs in the 3-5 minute range so I wouldn't go lower than 10-15m here if it's just intended to catch true runaway/hung jobs.  Anything under that could result in false positives if github's infra is slowed down.